import-export-cli/cmd/mi/mi.go

```diff
@@ -29,6 +33,10 @@ import (
 	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
 )
 
+var verbose bool
+var cfgFile string
+var insecure bool
+
 const miCmdShortDesc = "Micro Integrator related commands"
 
 const miCmdLongDesc = `Micro Integrator related commands such as login, logout, get, add, update, delete, activate, deactivate.`
```

Comment:
If this is ran as mi cli, better to have this updated as `mi is a Command Line Tool for managing WSO2 Micro Integrator`